src/sdk.ts

```diff
@@ -237,7 +237,7 @@ export class DefaultRelayingServices implements RelayingServices {
             tokenAddress,
             tokenAmount
         });
-        tokenAmount = tokenAmount ?? 0;
+        // tokenAmount = tokenAmount ?? 0;
```

Comment:
this param is optional, right? are we sure we want to remove this assignment? 

if `undefined`, won't line `277` cause an error?

Reply:
No, it won't. The following statement `a?.toString()` is similar to `a ?: a.toString():a` with some small differences

Reply:
my bad, tried it on the javascript console and it gave me an error, i suppose it's a ts thing :stuck_out_tongue: